clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -4777,6 +4782,9 @@ void SYCLIntegrationHeader::emit(raw_ostream &O) {
     O << "    return 0;\n";
     O << "#endif\n";
     O << "  }\n";
+    O << "  __SYCL_DLL_LOCAL\n";
+    O << "  static constexpr unsigned long getSize() { return " << K.ObjSize
```

Comment:
I'm not sure, but maybe we need to mention somewhere that the size is bytes? We can do that by including this info into function name or adding a comment to int header source.

Reply:
Maybe we should also somehow mention that it is size of a kernel object.

Reply:
> perhaps `getKernelSizeof`? `sizeof` is the operator name.

Okay, changed to getKernelSizeof.